Workflow failure: Release Build

## Workflow Failure

[Release Build](https://github.com/pulumi/pulumi-docker-containers/blob/main/.github/workflows/release.yml) has failed. See the list of failures below:

- [2023-07-20T23:14:07.000Z](https://github.com/pulumi/pulumi-docker-containers/actions/runs/5616659552)

- [2023-07-21T17:14:36.000Z](https://github.com/pulumi/pulumi-docker-containers/actions/runs/5616659552)

- [2023-07-26T19:46:54.000Z](https://github.com/pulumi/pulumi-docker-containers/actions/runs/5672918368)

Testing `UBI SDK images (java)` failed.

```
    environment.go:96: Created new test environment:  /tmp/test-env1786787818
    containers_test.go:68: Running command pulumi new java -y -f -s pulumi-test/container-java-1773f15c6f99e766
    containers_test.go:68: Ran command pulumi args [new java -y -f -s pulumi-test/container-java-1773f15c6f99e766] and expected success. Instead got failure.
    containers_test.go:68: Run Error: exit status 255
    containers_test.go:68: STDOUT: Created project 'test-env1786787818'
        
        Created stack 'pulumi-test/container-java-1773f15c6f99e766'
        
        ---------------------------------------------------
        constituent[0]: file:/root/.sdkman/candidates/maven/current/conf/logging/
        constituent[1]: file:/root/.sdkman/candidates/maven/current/lib/maven-model-builder-3.9.3.jar
        constituent[2]: file:/root/.sdkman/candidates/maven/current/lib/maven-core-3.9.3.jar
        constituent[3]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-provider-3.9.3.jar
        constituent[4]: file:/root/.sdkman/candidates/maven/current/lib/maven-model-3.9.3.jar
        constituent[5]: file:/root/.sdkman/candidates/maven/current/lib/failureaccess-1.0.1.jar
        constituent[6]: file:/root/.sdkman/candidates/maven/current/lib/httpcore-4.4.16.jar
        constituent[7]: file:/root/.sdkman/candidates/maven/current/lib/guava-31.1-jre.jar
        constituent[8]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-api-1.9.13.jar
        constituent[9]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-transport-http-1.9.13.jar
        constituent[10]: file:/root/.sdkman/candidates/maven/current/lib/javax.annotation-api-1.3.2.jar
        constituent[11]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-transport-wagon-1.9.13.jar
        constituent[12]: file:/root/.sdkman/candidates/maven/current/lib/commons-cli-1.5.0.jar
        constituent[13]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-named-locks-1.9.13.jar
        constituent[14]: file:/root/.sdkman/candidates/maven/current/lib/maven-repository-metadata-3.9.3.jar
        constituent[15]: file:/root/.sdkman/candidates/maven/current/lib/commons-codec-1.11.jar
        constituent[16]: file:/root/.sdkman/candidates/maven/current/lib/httpclient-4.5.14.jar
        constituent[17]: file:/root/.sdkman/candidates/maven/current/lib/maven-embedder-3.9.3.jar
        constituent[18]: file:/root/.sdkman/candidates/maven/current/lib/maven-plugin-api-3.9.3.jar
        constituent[19]: file:/root/.sdkman/candidates/maven/current/lib/maven-settings-3.9.3.jar
        constituent[20]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-transport-file-1.9.13.jar
        constituent[21]: file:/root/.sdkman/candidates/maven/current/lib/guice-5.1.0.jar
        constituent[22]: file:/root/.sdkman/candidates/maven/current/lib/wagon-file-3.5.3.jar
        constituent[23]: file:/root/.sdkman/candidates/maven/current/lib/javax.inject-1.jar
        constituent[24]: file:/root/.sdkman/candidates/maven/current/lib/plexus-cipher-2.0.jar
        constituent[25]: file:/root/.sdkman/candidates/maven/current/lib/maven-settings-builder-3.9.3.jar
        constituent[26]: file:/root/.sdkman/candidates/maven/current/lib/maven-artifact-3.9.3.jar
        constituent[27]: file:/root/.sdkman/candidates/maven/current/lib/aopalliance-1.0.jar
        constituent[28]: file:/root/.sdkman/candidates/maven/current/lib/maven-compat-3.9.3.jar
        constituent[29]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-util-1.9.13.jar
        constituent[30]: file:/root/.sdkman/candidates/maven/current/lib/slf4j-api-1.7.36.jar
        constituent[31]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-impl-1.9.13.jar
        constituent[32]: file:/root/.sdkman/candidates/maven/current/lib/org.eclipse.sisu.plexus-0.3.5.jar
        constituent[33]: file:/root/.sdkman/candidates/maven/current/lib/wagon-provider-api-3.5.3.jar
        constituent[34]: file:/root/.sdkman/candidates/maven/current/lib/plexus-component-annotations-2.1.0.jar
        constituent[35]: file:/root/.sdkman/candidates/maven/current/lib/jansi-2.4.0.jar
        constituent[36]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-spi-1.9.13.jar
        constituent[37]: file:/root/.sdkman/candidates/maven/current/lib/jcl-over-slf4j-1.7.36.jar
        constituent[38]: file:/root/.sdkman/candidates/maven/current/lib/maven-shared-utils-3.3.4.jar
        constituent[39]: file:/root/.sdkman/candidates/maven/current/lib/commons-lang3-3.12.0.jar
        constituent[40]: file:/root/.sdkman/candidates/maven/current/lib/plexus-sec-dispatcher-2.0.jar
        constituent[41]: file:/root/.sdkman/candidates/maven/current/lib/plexus-interpolation-1.26.jar
        constituent[42]: file:/root/.sdkman/candidates/maven/current/lib/maven-resolver-connector-basic-1.9.13.jar
        constituent[43]: file:/root/.sdkman/candidates/maven/current/lib/wagon-http-3.5.3.jar
        constituent[44]: file:/root/.sdkman/candidates/maven/current/lib/plexus-utils-3.5.1.jar
        constituent[45]: file:/root/.sdkman/candidates/maven/current/lib/maven-builder-support-3.9.3.jar
        constituent[46]: file:/root/.sdkman/candidates/maven/current/lib/maven-slf4j-provider-3.9.3.jar
        constituent[47]: file:/root/.sdkman/candidates/maven/current/lib/wagon-http-shared-3.5.3.jar
        constituent[48]: file:/root/.sdkman/candidates/maven/current/lib/org.eclipse.sisu.inject-0.3.5.jar
        ---------------------------------------------------
    containers_test.go:68: STDERR: Logging in using access token from PULUMI_ACCESS_TOKEN
        Exception in thread "main" java.lang.Error: java.io.FileNotFoundException: /usr/lib/jvm/java-11-openjdk-11.0.20.0.8-2.el8.x86_64/lib/tzdb.dat (No such file or directory)
        	at java.base/sun.util.calendar.ZoneInfoFile$1.run(ZoneInfoFile.java:261)
        	at java.base/sun.util.calendar.ZoneInfoFile$1.run(ZoneInfoFile.java:251)
        	at java.base/java.security.AccessController.doPrivileged(Native Method)
        	at java.base/sun.util.calendar.ZoneInfoFile.<clinit>(ZoneInfoFile.java:251)
        	at java.base/sun.util.calendar.ZoneInfo.getTimeZone(ZoneInfo.java:588)
        	at java.base/java.util.TimeZone.getTimeZone(TimeZone.java:577)
        	at java.base/java.util.TimeZone.setDefaultZone(TimeZone.java:682)
        	at java.base/java.util.TimeZone.getDefaultRef(TimeZone.java:653)
        	at java.base/java.util.TimeZone.getDefault(TimeZone.java:642)
        	at java.base/java.util.Calendar.defaultTimeZone(Calendar.java:1679)
        	at java.base/java.util.Calendar.getInstance(Calendar.java:1660)
        	at java.base/java.text.SimpleDateFormat.initializeCalendar(SimpleDateFormat.java:676)
        	at java.base/java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:620)
        	at org.eclipse.sisu.plexus.PlexusDateTypeConverter.<clinit>(PlexusDateTypeConverter.java:37)
        	at org.codehaus.plexus.DefaultPlexusContainer$DefaultsModule.<init>(DefaultPlexusContainer.java:851)
        	at org.codehaus.plexus.DefaultPlexusContainer.<init>(DefaultPlexusContainer.java:147)
        	at org.apache.maven.cli.MavenCli.container(MavenCli.java:648)
        	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:277)
        	at org.apache.maven.cli.MavenCli.main(MavenCli.java:206)
        	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:283)
        	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:226)
        	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:407)
        	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:348)
        Caused by: java.io.FileNotFoundException: /usr/lib/jvm/java-11-openjdk-11.0.20.0.8-2.el8.x86_64/lib/tzdb.dat (No such file or directory)
        	at java.base/java.io.FileInputStream.open0(Native Method)
        	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
        	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
        	at java.base/sun.util.calendar.ZoneInfoFile$1.run(ZoneInfoFile.java:255)
        	... 26 more
        error: installing dependencies failed; rerun manually to try again, then run `pulumi up` to perform an initial deployment: '/root/.sdkman/candidates/maven/current/bin/mvn /root/.sdkman/candidates/maven/current/bin/mvn -Dorg.slf4j.simpleLogger.defaultLogLevel=warn --no-transfer-progress compile' exited with non-zero exit code: 1
```

--- a/docker/java/Dockerfile.ubi
+++ b/docker/java/Dockerfile.ubi
@@ -26,7 +26,7 @@ RUN microdnf install -y \
     zip \
     findutils \
     ca-certificates \
-    tzdata \
+    tzdata-java \
     && microdnf clean all && [ ! -d /var/cache/yum ] || rm -rf /var/cache/yum
 
 RUN curl -fsSL "https://get.sdkman.io" | bash \

--- a/docker/java/Dockerfile.ubi
+++ b/docker/java/Dockerfile.ubi
@@ -26,6 +26,7 @@ RUN microdnf install -y \
     zip \
     findutils \
     ca-certificates \
+    tzdata \
     && microdnf clean all && [ ! -d /var/cache/yum ] || rm -rf /var/cache/yum
 
 RUN curl -fsSL "https://get.sdkman.io" | bash \

